### PR TITLE
/api/srch/profiles: search only by user tag 

### LINKED ISSUE #3506: Profiles search page: search by user tag
Right now, the Profiles option on the search page (https://publiclab.org/search/profiles/balloons/) is searching for users by username and bio.

In the endpoint, we would like to add the option to also search for user tags. [Here](https://github.com/publiclab/plots2/blob/master/app/api/srch/search.rb#L85) we already have the parameter/value `field=username`, it is used when we want to search only by username. One option would be to add `field=tag`.

### PR DESCRIPTION
**Pull request by milaaraujo**
Fixes #3506

/api/srch/profiles: Adding the option to search for user tags.

`/api/srch/profiles?query=[tag_name]&field=tag`

### CODE REVIEW

File: doc/API.md [hunk 1]
```diff
@@ -43,9 +43,7 @@ All the endpoints have the optional parameter `limit` (10 by default) where you
   `order_direction=[string]`: It accepts `ASC` or `DESC` (the latter is the default).
 
   `field=[string]`: Accepts the value `username` for searching profiles only by
-   the field `username`.
-
-   `tag=[string]`: The search can be refined by passing a tag field.
+   the field `username`. And accepts the value `tag` for searching profiles by tag.
```
Comment (jywarren) [inline doc/API.md]: ah but the value `tag` is removed now, no? Or maybe I mis-read?

**Comment by milaaraujo** [comment]
I removed the ‘tag’ option. But to search by tag it is necessary to send the *value* “tag” in the option ‘field’: ‘/api/srch/profiles?query=[tag_name]&field=tag’

In the same way that it is necessary to send  ‘/api/srch/profiles?query=[search_query]&field=username’ to search only by username (instead of username and bio).

Make sense?

**Comment by jywarren** [comment]
Ah, yes, i see. Great. Merging!

**Comment by jywarren** [comment]
And can we make sure that option is enabled in the profile search results page by default? Thank you!

**Comment by milaaraujo** [comment]
>And can we make sure that option is enabled in the profile search results page by default? Thank you!

I think this is what @cesswairimu is working on (3911)!

**Comment by jywarren** [comment]
oh awesome. This is all coming together!

**Pull request merged**